packages/dds/tree/src/shared-tree-core/branch.ts

```diff
@@ -656,3 +646,63 @@ export function onForkTransitive<T extends ISubscribable<{ fork: (t: T) => void
 	);
 	return () => offs.forEach((off) => off());
 }
+
+class RevertibleRevision implements Revertible {
```

Comment:
this doesn't seem to be specific to shared tree at all so maybe it should be moved to the core revertible package?

Reply:
Right now, it is an implementation detail of Branch so I'd rather not put it in another file where it will need to be exported thus allowing other parts of the code to depend on it.